lib/OrdinaryDiffEqCore/src/alg_utils.jl

```diff
@@ -412,3 +412,12 @@ is_mass_matrix_alg(alg::Union{OrdinaryDiffEqAlgorithm, DAEAlgorithm}) = false
 is_mass_matrix_alg(alg::CompositeAlgorithm) = all(is_mass_matrix_alg, alg.algs)
 is_mass_matrix_alg(alg::RosenbrockAlgorithm) = true
 is_mass_matrix_alg(alg::NewtonAlgorithm) = !isesdirk(alg)
+
+# All algorithms should be shown using their keyword definition, and not as structs
+function Base.show(io::IO, alg::OrdinaryDiffEqAlgorithm)
+    print(io, String(typeof(alg).name.name), "(;")
+    for fieldname in fieldnames(typeof(alg))
+        print(io, " ", fieldname, " = ", getfield(alg, fieldname), ",")
+    end
+    print(io, ")")
+end
```

Comment:
Should this be further upstreamed/generalized to SciMLAlgorithm?

Reply:
If it'scorrect? I don't think it is as a 2-arg show. I think it needs to be on the 3-arg with a text mimetype since I don't think it round trips.

Reply:
This is the `show` method that is used in the latest release.
This commit just moves it from a wrong location in OrdinaryDiffEqLowStorageRK to OrdinaryDiffEqCore.

I could swap it to `Base.show(io::IO, ::MIME"text/plain", alg::OrdinaryDiffEqAlgorithm)`, I have no strong opinions.

Reply:
That dispatch would be better.